pkg/config/config.go

```diff
@@ -75,6 +75,8 @@ var (
 	ExposeVMStats                = getBoolConfig("EXPOSE_VM_METRICS", true)
 	ExposeHardwareCounterMetrics = getBoolConfig("EXPOSE_HW_COUNTER_METRICS", true)
 	ExposeIRQCounterMetrics      = getBoolConfig("EXPOSE_IRQ_COUNTER_METRICS", true)
+	ExposeBPFMetrics             = getBoolConfig("EXPOSE_BPF_METRICS", true)
```

Comment:
Is it worth considering making these `false` by default?
Default config would be just these: https://sustainable-computing.io/design/metrics/#kepler-metrics-for-container-energy-consumption

Reply:
Agree that the operator can provide a way to tweak the settings.
Deployment of kepler by operator can turn this off by default.